src/floats.lisp

```diff
@@ -0,0 +1,52 @@
+;;;; floats.lisp
+;;;;
+;;;; Author: Robert Smith
+
+(in-package #:qvm)
+
+;;; This file deals with the different floating point representations
+;;; made use of in the QVM.
+;;;
+;;; These are agreed upon *everywhere* in the QVM. These are allowed
+;;; to change, but only here.
+
+(defconstant +octets-per-flonum+ 8)
+
+(deftype flonum (&optional (min '*))
```

Comment:
`deftype` optionals default to `*` already.

Reply:
New fact; did not know!